### ISSUE TITLE
Update for Circe 0.12 and diffson 4.0.0

### ISSUE THREAD

**Issue by koterpillar** [stephennancekivell/scalatest-json#28]
Running with Circe 0.12 produces either

```
java.lang.NoSuchMethodError: io.circe.Encoder$.encodeList(Lio/circe/Encoder;)Lio/circe/ArrayEncoder;
```

or

```
java.lang.NoClassDefFoundError: gnieh/diffson/circe$
```

**Comment by stephennancekivell** [comment]
what version of scalatest-circe are you using. I just published 0.0.2 which supports circe 0.12

**Comment by koterpillar** [comment]
I have indeed overlooked the 0.0.2 version, it works well now.
